obal/data/modules/srpm.py

```diff
@@ -5,8 +5,15 @@
 import shutil
 import os
 import subprocess
+from zipfile import ZipFile
 from contextlib import contextmanager
-from tempfile import mkdtemp
+from tempfile import mkdtemp, TemporaryFile
+
+try:
+    from urllib.request import urlopen
+    from urllib.error import HTTPError
+except ImportError:
+    from urllib2 import urlopen, HTTPError
```

Comment:
Ansible vendors [`six`](https://six.readthedocs.io/), and that has [`six.moves`](https://six.readthedocs.io/#module-six.moves) to take care of these games.

So you could do smth like:

```python
from ansible.module_utils.six.moves.urllib.request import urlopen
from ansible.module_utils.six.moves.urllib.error import HTTPError
```

(untested)

But you don't have to, just a comment ;)